lib/data/landing_page_content_items/tasks.yaml

```diff
@@ -149,6 +149,23 @@ blocks:
             csv_file: "data_one.csv"
             data_source_link: https://www.ons.gov.uk/economy/grossdomesticproductgdp/timeseries/ihyq/qna
             padding: true
+- type: two_column_layout
+  theme: two_thirds_one_third
+  blocks:
+  - type: press_notices
+    items:
+    - text: An example title for this press notice 1
```

Comment:
Might need to check with @richardTowers on his plans for how content is pulled into this space, although this looks fine for now.